buzzmoon-frontend/buzzmoon-vite/src/Components/Results/Results.jsx

```diff
@@ -54,16 +59,43 @@ export default function Results(props) {
         </Heading>
         <Tabs w={{base:"95vw", sm: "95vw", md:"100%"}} size='md' variant='enclosed' colorScheme={'black'}>
         <TabList>
-          <Tab>Standard</Tab>
-          <Tab>Head 2 Head</Tab>
-          <Tab>Best Buzzes</Tab>
+          <Tab>
+          <HStack align={'start'}>
+            <Text>Standard</Text>
+            <Tooltip hasArrow
+              label='Players are ranked by points scored'
```

Comment:
nit: use function that takes in label, bg color, color and return the tooltip component, just for extra cleanup